internal/checker/terraform/rules/gcp/google_storage_bucket_uniform_bucket_level_access_enabled.rego

```diff
@@ -0,0 +1,54 @@
+# METADATA
+# title: "Verify Uniform Bucket-Level Access for Cloud Storage Buckets"
+# description: "This policy ensures that Cloud Storage buckets have uniform bucket-level access enabled, promoting consistent and secure access controls across the buckets."
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket
+# custom:
+#   id: CB_TFGCP_038
+#   severity: MEDIUM
+package lib.terraform.CB_TFGCP_038
+
+import future.keywords.in
+
+isvalid(resource) {
+	resource.Type == "resource"
+	"google_storage_bucket" in resource.Labels
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
```

Comment:
you should check pass instead because the default value of "uniform_bucket_level_access" is false

check uniform_bucket_level_access == true to pass